scripts/create2/get-signer.js

```diff
@@ -0,0 +1,45 @@
+const { ethers } = require('hardhat');
+const { AwsKmsSigner } = require('@nexusmutual/ethers-v5-aws-kms-signer');
+
+const SIGNER_TYPE = {
+  LOCAL: 'local',
+  AWS_KMS: 'aws-kms',
+};
+
+const getSigner = async (kind = SIGNER_TYPE.LOCAL) => {
+  if (kind === SIGNER_TYPE.LOCAL) {
+    const [signer] = await ethers.getSigners();
+    return signer;
+  }
+
+  const provider = ethers.provider;
+  const { AWS_KMS_KEY_ID, AWS_REGION } = process.env;
+
+  if (kind === SIGNER_TYPE.AWS_KMS) {
+    return new AwsKmsSigner(AWS_KMS_KEY_ID, AWS_REGION, provider);
+  }
+
+  throw new Error(`Invalid signer type: ${kind}`);
+};
+
+module.exports = { SIGNER_TYPE, getSigner };
+
+if (require.main === module) {
+  (async () => {
+    // get signer
+    const signer = await getSigner(SIGNER_TYPE.AWS_KMS);
+    const address = await signer.getAddress();
+    console.log(`Signer address: ${address}`);
+
+    // send tx to self
+    const tx = await signer.sendTransaction({
+      to: address,
+      value: '0',
+      maxFeePerGas: ethers.utils.parseUnits('15', 'gwei'),
+      maxPriorityFeePerGas: ethers.utils.parseUnits('1', 'gwei'),
+    });
+    console.log(`Tx sent: https://etherscan.io/tx/${tx.hash}`);
```

Comment:
what is this for? is it to test whether the given `AWS_KMS_KEY_ID` and  `AWS_REGION` values work?

Reply:
yes, that's what I used to test it. a bit unorthodox I suppose. I'm aware one of the tests isn't running because of missing env keys, what was the test plan for that?

Reply:
`getAddress` should be enough to verify `AWS_KMS_KEY_ID` and `AWS_REGION` are correct w/o spending gas.

Reply:
do you mean the `awsKmsSigner.spec.ts`? its an e2e test so it needs env vars, its there if we want to test e2e but it skips if its not provided.

Reply:
I've added this section to its README for clarity:

## Tests

```
npm test
```

To run end to end test set the following env vars:
* `TEST_KMS_REGION_ID` - The AWS region where the KMS key is located
* `TEST_KMS_KEY_ID` - The ID of the AWS KMS key

Reply:
Great, thanks!